src/libs/ReportUtils.js

```diff
@@ -2627,7 +2627,7 @@ function buildTransactionThread(reportAction, moneyRequestReportID) {
         participantAccountIDs,
         getTransactionReportName(reportAction),
         '',
-        lodashGet(getReport(reportAction.reportID), 'policyID', CONST.POLICY.OWNER_EMAIL_FAKE),
```

Comment:
What's this change for?

Reply:
no idea where this is coming from tbh.. just removed it